ui/webui/src/app.jsx

```diff
@@ -46,6 +53,46 @@ export const Application = () => {
         return null;
     }
 
+    const wrapWithContext = children => {
+        return (
+            <AddressContext.Provider value={address}>
+                {children}
+            </AddressContext.Provider>
+        );
+    };
+
+    const steps = [
+        {
+            name: _("Installation language"),
+            component: wrapWithContext(<InstallationLanguage />),
+            stepNavItemProps: { id: "installation-language" }
+        },
+        {
+            name: _("Storage configuration"),
+            component: wrapWithContext(<InstallationDestination />),
+            stepNavItemProps: { id: "installation-destination" }
+        },
+        {
+            name: _("Review"),
+            component: wrapWithContext(<ReviewConfiguration />),
+            nextButtonText: _("Begin installation"),
+            stepNavItemProps: { id: "review-configuration" }
+        },
+        {
+            name: _("Installation progress"),
+            component: wrapWithContext(<InstallationProgress />),
```

Comment:
This uses the onAddErrorNotification handler. Please add the property like: 
```
InstallationProgress onAddNotification={onAddNotification} />
```

Reply:
Added. :)